proto/protovalidate-testing/buf/validate/conformance/cases/oneofs.proto

```diff
@@ -66,8 +75,8 @@ message OneofIgnoreEmpty {
     ];
     int32 z = 3 [
       (buf.validate.field).int32 = {
-        lte: 128,
-        gte: 256,
+        gte: 128,
+        lte: 256,
```

Comment:
FWIW, it was legal the other way, too. Alfred was explaining this to me the other day, that when the low and high are flipped, it effectively enforces that the value is _not_ in the range in between (i.e. "gte" and "lte" end up combined with OR instead of AND).

Reply:
That's correct, however in context of testing the `ignore_empty` constraint, having it the other way around made the `ignore_empty` uninteresting (0 would always be valid in the constraints)